hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -152,18 +152,17 @@
       { name: "lc_usb",         gen: true,  type: "top", parent: "lc_src",  clk: "usb"     }
       { name: "sys",            gen: true,  type: "top", parent: "sys_src", clk: "main"    }
```

Comment:
i'm not completely sure we need the `sys` and `sys_io` resets anymore, but i could be wrong.

Reply:
sys is connected to rv_dm instead of lc. But, sys_io and sys_aon are not used so I removed them.